Topics/Tech_Stacks/Flutter.md

```diff
@@ -0,0 +1,49 @@
+# Learning Flutter
+
+## Table of contents
+
+#### [What is Flutter](#what-is-flutter)
+
+#### [Creating your first flutter app](#creating-your-first-flutter-app)
+
+## What is Flutter?
+
+Flutter is an open source framework developed and supported by Google. Frontend and full-stack developers use Flutter to build an application’s user interface (UI) for multiple platforms with a single codebase.
+
+Flutter now supports application development on six platforms: iOS, Android, the web, Windows, MacOS, and Linux.
+
+Flutter uses the open-source programming language Dart, which was also developed by Google. Dart is optimized for building UIs, and many of Dart’s strengths are used in Flutter.
+
+Source: [Amazon AWS](https://aws.amazon.com/what-is/flutter/)
+
+## Creating your first flutter app
+
+The following is a summary from the offical [Flutter Docs](https://docs.flutter.dev/get-started/).
+
+1. **Install Flutter**
+   a) [Windows](https://docs.flutter.dev/get-started/install/windows)
```

Comment:
This displays in-line instead of in a list. To make it more readable, you can use a numbered list, which is natively supported by Markdown. Otherwise, for a lettered list, you can just embed HTML in your file, like this:

```
<ol type="a">
  <li>Coffee</li>
  <li>Tea</li>
  <li>Milk</li>
</ol>
```

Which will compile to this:

<ol type="a">
  <li>Coffee</li>
  <li>Tea</li>
  <li>Milk</li>
</ol>